vspace/src/main/webapp/WEB-INF/views/exhibition/module.html

```diff
@@ -11,7 +11,7 @@
    </head>
    <body>
     <div layout:fragment="content">
-        <div th:if="${showAlert == true}"
+        <div th:if="${showAlert == true AND (isExhPreview != true)}"
```

Comment:
basically the whole div is the same except the link right? Just add an the if statement into the expression fo the href attribute instead of repeating the whole block, e.g. `${teacher.active} ? 'ACTIVE' : 'RETIRED'`

Reply:
this goes for all the templates and expressions